doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -312,6 +312,14 @@ Bluetooth samples
   * Updated by simplifying the 2-wire UART polling.
     This is done by replacing the hardware timer with the ``k_sleep()`` function.
 
+* Disabled legacy pairing from the following samples:
+
+   * :ref:`central_nfc_pairing`
+   * :ref:`peripheral_power_profiling`
+
+* Support for legacy pairing remains exclusively for :ref:`peripheral_nfc_pairing` sample,
+  to retain compatibility with older Andorid devices.
```

Comment:
```suggestion
* Disabled legacy pairing from the following samples:

   * :ref:`central_nfc_pairing`
   * :ref:`peripheral_power_profiling`

   Support for legacy pairing remains exclusively for :ref:`peripheral_nfc_pairing` sample to retain compatibility with older Andorid devices.
```
A better place for this entry might be a few lines up right above the DTM sample entry.